sparqlbuilder/src/main/java/org/eclipse/rdf4j/sparqlbuilder/rdf/Rdf.java

```diff
@@ -155,56 +199,96 @@ public static RdfPredicateObjectList predicateObjectList(RdfPredicate predicate,
 	/**
 	 * Create a {@link RdfPredicateObjectListCollection} with an initial {@link RdfPredicateObjectList}
 	 * 
-	 * @param predicate the {@link RdfPredicate} of the initial {@link RdfPredicateObjectList}
-	 * @param objects the {@link RdfObject}(s) of the initial {@link RdfPredicateObjectList}
-	 * 
+	 * @param predicate
+	 *        the {@link RdfPredicate} of the initial {@link RdfPredicateObjectList}
+	 * @param objects
+	 *        the {@link RdfObject}(s) of the initial {@link RdfPredicateObjectList}
 	 * @return a new {@link RdfPredicateObjectListCollection}
 	 */
-	public static RdfPredicateObjectListCollection predicateObjectListCollection(RdfPredicate predicate, RdfObject... objects) {
+	public static RdfPredicateObjectListCollection predicateObjectListCollection(RdfPredicate predicate,
+			RdfObject... objects)
+	{
 		return new RdfPredicateObjectListCollection().andHas(predicate, objects);
 	}
 
 	/**
 	 * Create a {@link RdfPredicateObjectListCollection} with the given {@link RdfPredicateObjectList}(s)
 	 * 
-	 * @param predicateObjectLists the {@link RdfPredicateObjectList}(s) to add to the collection
-	 * 
+	 * @param predicateObjectLists
+	 *        the {@link RdfPredicateObjectList}(s) to add to the collection
 	 * @return a new {@link RdfPredicateObjectListCollection}
 	 */
-	public static RdfPredicateObjectListCollection predicateObjectListCollection(RdfPredicateObjectList... predicateObjectLists) {
+	public static RdfPredicateObjectListCollection predicateObjectListCollection(
+			RdfPredicateObjectList... predicateObjectLists)
+	{
 		return new RdfPredicateObjectListCollection().andHas(predicateObjectLists);
 	}
-	
+
 	/**
-	 * Convert an array of {@link String}s to an array of {@link StringLiteral}s
+	 * Converts an array of object {@link Value}s to an array of {@link RdfObject}s.
 	 * 
-	 * @param literals the {@link String}s to convert
+	 * @param values
+	 *        an array of {@link Value}.
+	 * @return an array of {@link RdfObject}
+	 */
+	public static RdfObject[] objects(Value... values) {
+		final RdfObject[] result = new RdfObject[values.length];
```

Comment:
I think it could be useful having a method that converts a single `Value` to a single `RdfObject` (so everything in the `for` loop) in case a user would like to do that and not get an array back. And then this could be one-lined using streams, something like `Arrays.stream(values).map(Rdf::convertValueToRdfObject).toArray(RdfObject[]::new)`